client/src/elements/schema-editor/schema-editor-files.js

```diff
@@ -191,7 +209,12 @@ export class SchemaEditorFiles {
         this.dropzone.files.push(mockFile);
         this.dropzone.emit("addedfile", mockFile);
         if (file.type && file.type.includes("image/")) {
-          mockFile.dataURL = file.url; // needed for dropzone to create the thumbnail in a canvas
+          const imageUrl = fileBaseUrl ? `${fileBaseUrl}/${file.key}` : file.url;
+          const response = await this.httpClient.fetch(imageUrl);
```

Comment:
How is the behavior here if the image fails to load?
I'd move that code before the `this.dropzone.files.push` call.

Reply:
Imo we want to have the image inside the dropzone even if the thumb generation (loading or other errors) fails here because we do have the image reference in the database on the item.

so not showing anything in case the loading fails is misleading imo.